### PR TITLE
Reverses Cult Nerfs + some QoL

### PR DESCRIPTION
**Pull request by Cenrus**
https://github.com/tgstation/tgstation/pull/50997

As it stands, blood cult isn't dangerous or threatening enough and this PR hopes to address that. 

While the nerfs have been reverted, cult barriers have been changed so they're not invincible, and it's no longer possible to use mass recall as the cult leader in summoning sites. These changes should make stopping an attempted summon much, much fairer for the crew.

:cl:
balance: cult barrier runes can be smashed until they break
balance: cult leaders can no longer use final reckoning in narsie summon locations
balance: Cult armor values buffed slightly
balance: Cult shield buffed
balance: Cult stun isn't blocked by mindshield anymore, don't fight them alone.
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
As much as I hate 1-click stun cult needs some improvements.
:+1:

**Comment by ghost** [comment]
Combat one-hit stuns aren't good, are you gonna give securitys the taser and old baton back and tell revolutionaries "well don't fight them alone" to justify that?

**Comment by Cenrus** [comment]
It's a nessecary evil, the stun is what makes the cult dominate in melee, as they should. They have no ranged options.

**Comment by ghost** [comment]
> It's a nessecary evil, the stun is what makes the cult dominate in melee, as they should. They have no ranged options.

They have bolas with no counter, constructs that fire magic missiles.

**Comment by february-ops** [comment]
> > It's a nessecary evil, the stun is what makes the cult dominate in melee, as they should. They have no ranged options.
> 
> They have bolas with no counter, constructs that fire magic missiles.

Construct has nearly any health killing is very easy ,bola can be stoped by having throw mode on, and if you have more than one person with you they can take the leg cuff of and you can outrun missiles

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	block_level = 0
 	block_flags = BLOCKING_ACTIVE | BLOCKING_NASTY
 	force = 15
-	throwforce = 12 // unlike normal daggers, this one is curved and not designed to be thrown
+	throwforce = 25
```
Comment (Archanial) [inline code/modules/antagonists/cult/cult_items.dm:25]: undocumented change smh

**Comment by Archanial** [comment]
> They have no ranged options.

This is not true, bolas are a thing and they are really good.

**Comment by ghost** [comment]
> > > It's a nessecary evil, the stun is what makes the cult dominate in melee, as they should. They have no ranged options.
> > 
> > 
> > They have bolas with no counter, constructs that fire magic missiles.
> 
> bola can be stoped by having throw mode on

Wrong, holding a cult bola hardstuns you.

**Comment by VictorVow** [comment]
No to the cult stun revert, having it one-shot any security in maint is cringe because it's so easy to ambush with it.

**Comment by Cenrus** [comment]
They need the stun to fight security, we have HoS players roaming maintenance alone on cult rounds because there's not a lot cult can do against them.

**Comment by ghost** [comment]
> They need the stun to fight security, we have HoS players roaming maintenance alone on cult rounds because there's not a lot cult can do against them.

One well placed bola and the Head of Security is done for, one EMP and everything besides their stunbaton is disabled.

**Comment by Archanial** [comment]
> 
> 
> > They need the stun to fight security, we have HoS players roaming maintenance alone on cult rounds because there's not a lot cult can do against them.
> 
> One well placed bola and the Head of Security is done for, one EMP and everything besides their stunbaton is disabled.

Not like stun baton is bad or anything? Besides, he still has fashbangs, bolas, pepper spray and shotgun.

**Comment by heepox** [comment]
just increase bola stun instead and give them cult bear traps that trigger on non-cult.

**Comment by VictorVow** [comment]
Sword blocks the baton, and by the time you're chopping up the bola'd seccie the other options won't matter. 

If there's a shotgun or other ballistics then the same advice for cult goes in reverse - don't engage them solo. 

Give the blindfold a secHUD so they can detect mindshielded people instead.

**Comment by PowerfulBacon** [comment]
Being full stunned in combat does suck a lot, it should give some confusion or stamina damage and not be a full on instant win. (fighting in groups doesn't help since the cultist can just get 2 stun hands)

**Comment by VictorVow** [comment]
Small stamina damage and dizziness could work, so if the stun is against mindshielded people then it's good for escaping them but not for taking them down. They should be able to see mindshields with their blindfold though so they don't waste their stun on a protected target.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
I don't like this since it brings back combat stuns which kind of suck. I think a layering system would be better where the cult stun is super effective against non mindshielded people, and does a weaker non stunning but still annoying effect (like mute) against minsdhielded people since combat stunning sucks. Either that or give the cultists a hud which shows mindshields similar to revs (Revs can see mindshields right?).

**Pull request closed without merging**